### DISCUSSION TITLE
Logs cannot be read in Web UI

### DISCUSSION THREAD

**Discussion by daniel1v** [nextcloud/all-in-one#3580] [Questions] [answered]
### Steps to reproduce
1. Open logs in web interface 

### Expected behavior
Logs should be shown

### Actual behavior
Spinning wheel, high cpu usage, logs are spammed massively with repeating error message until container restart. Closing Browser window does not help. 

### Host OS 
Debian bookworm

#### Nextcloud AIO version 
7.4.1

#### Current channel 
Release

#### Other valuable info
This log entry appears repeatedly:

NOTICE: PHP message: {"reqId":"VKDLv6SDzeEUCtDg6qOK","level":3,"tim
e":"2023-10-21T06:59:11+00:00","remoteAddr":"192.168.178.xx","user"
:"xxx","app":"PHP","method":"GET","url":"/apps/logreader/get?offse
t=0&count=50&levels=11111","message":"fread(): Read of 8192 bytes f
ailed with errno=21 Is a directory at /var/www/html/apps/logreader/
lib/Log/LogIterator.php#102","userAgent":"Mozilla/5.0 (Linux; Andro
id 10; K) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/117.0.0.0 M
obile Safari/537.36 EdgA/117.0.2045.65","version":"27.1.2.1","data"
:{"app":"PHP"}}

**Comment by szaimen** [comment]
Hi, I wonder what causes this high cpu usage. Can you disable all optional addons from the aio interface and try again?

**Comment by daniel1v** [comment, marked as answer]
hi, @szaimen and thank you for your reply.

unfortunately it seems, I was not clear enough in my description: the high cpu usage started, as soon as I tried to access the nextcloud log file via the nextcloud web interface. the ui wheel kept spinning indefinitely and the docker logs got spammed with the above error message. so it had nothing to do with the other addons.

in the meantime, I was able to do some research by myself and checked my nextcloud log inside the docker aio volume. it turned out, that nextcloud.log was a folder and not a file. this led to the issues described above. after deleting the nextcloud.log folder and restarting containers, everything now seems to work.

seems like it was my bad, I mounted the logfile into my crowdsec container and maybe did something wrong there.
